packages/components/src/DataViewer/ModelViewer/__snapshots__/ModelViewer.container.test.js.snap

```diff
@@ -1,50 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`I18n should render 1`] = `
-<ModelViewerContainer
```

Comment:
The previous snapshot was useless, it doesn't test anything. The test asjk to render a `ModelViewerContainer` and the snapshot shows a ... 🥁 ... `ModelViewerContainer`

Explanation: the `ModelViewerContainer` is wrapped with i18n HOC, so what we see is what the HOC is supposed to render.

The test has not changed, but the i18n HOC mock is now the original component (with name and default props.t changed). It renders the real thing underneath: the TreeManager with default componentId